src/Host/Protocol/Impl/CustomHttpError.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.R.Host.Protocol {
+    public enum CustomHttpError {
```

Comment:
I didn't want to use strings since broker locale may be different from user machine locale(s). Each user can have different locale set, so numbers work better.